sync-github-vsts.yml

```diff
@@ -0,0 +1,41 @@
+# variables:
+  # ADOUrl: defined on azure portal
+
+trigger:
+  batch: true
+  branches:
+    include:
+    - v3
+    - v3-release
+    - dev
+    - main
+
+variables:
+  NugetSecurityAnalysisWarningLevel: none
+  
+jobs:
+- job: 'Sync_GitHub_To_AzureDevOps'
+  pool: 
+    vmImage: 'windows-latest'
+  steps:
+  - task: Powershell@2
+    displayName: sync from GitHub to Azure DevOps
+    env:
+      SYSTEM_ACCESSTOKEN: $(System.AccessToken)
+    inputs:
+      targetType: inline
+      script: |
+        Function Sync([string]$branch) {
+          git fetch $(Build.Repository.Uri) $branch
+          git checkout FETCH_HEAD
+          git checkout -b $branch
```

Comment:
should the repository already been implicitly checked out by azure pipeline? All we need is a `git push`?

Reply:
Not really, you need to checkout the specific branch firstly